api.go

```diff
@@ -104,6 +108,8 @@ type ProtocolParameters interface {
 	NetworkName() string
 	// NetworkID defines the ID of the network which is derived from the network name.
 	NetworkID() NetworkID
+	// SlotsPerEpochExponent defines the exponent used to calculate the number of slots per epoch.
+	SlotsPerEpochExponent() uint8
```

Comment:
This seems lost here. Shouldn't it be in the TimeProvider?

Reply:
if we really need this exposed, then please move it below to the level of the `TimeProvider()`